src/bin/pg_autoctl/ipaddr.c

```diff
@@ -79,31 +128,23 @@ fetchLocalIPAddress(char *localIpAddress, int size,
 		}
 		else
 		{
-			log_warn("Failed to connect: %m");
+			log_warn("Failed to connect to any of the IP addresses for "
+					 "monitor hostname \"%s\" and port %d",
```

Comment:
8.8.8.8 is not a monitor
```suggestion
					 "hostname \"%s\" and port %d",
```

Reply:
Good point again. Also when given a dotted IP address then `getaddrinfo` does not actually do a DNS query to return several IP addresses, it only does the parsing and prepares the needed binary structure. I have now added a specific message to the case of using `DEFAULT_INTERFACE_LOOKUP_SERVICE_NAME`.